src/osp/coordinates.h

```diff
@@ -52,14 +52,25 @@ struct CCompSat { using datatype_t = Satellite; };
 template <typename CCOMP_T>
 using ViewCComp_t = Corrade::Containers::StridedArrayView1D<typename CCOMP_T::datatype_t>;
 
+using CoordinateView_t = Corrade::Containers::StridedArrayView1D<void>;
+
+using coordspace_index_t = uint32_t;
+
+/**
+ * Runtime-generated sequential ID for a CComp
+ */
 template<typename CCOMP_T>
-constexpr ccomp_id_t ccomp_id() noexcept
+inline ccomp_id_t ccomp_id = ccomp_family_t::type<CCOMP_T>;
```

Comment:
Wouldn't you want this to be constexpr?

Reply:
Putting constexpr here caused a compile error. The type variable itself isn't constexpr, and is incremented at runtime.

Reply:
Ah. Hrmmm.

That could potentially mean a race-condition across multiple threads. But I haven't thought about it carefully.

Reply:
I think it probably does; EnTT has an option to make families atomic